software/common/common.sh

```diff
@@ -69,7 +69,7 @@ The following options are available:
                 [--cov]       - generate coverage reports
   cov_upload  Upload coverage reports to Codecov server
   unit        Builds and runs unit tests only (and generates coverage reports)
-                  <name>  - run specific unit test, may include wildcards, i.e. 'debug*'
+                  <name>  - run specific unit test, may include wildcards, i.e. '*common*'
```

Comment:
Why this change?

Reply:
Filters now need to have this form to work, as unit test names usually starts with `test_`. I used `*common*` instead of `*debug*` because I wanted to give an example that actually works when using this script. Since then, the tests were restructured, and this example no longer works, will replace it with one that actually works.